std/numeric.d

```diff
@@ -79,10 +79,12 @@ public enum CustomFloatFlags
     none = 0
 }
 
+private enum isIEEEQuadruple = floatTraits!real.realFormat == RealFormat.ieeeQuadruple;
+
 private template CustomFloatParams(uint bits)
 {
     enum CustomFloatFlags flags = CustomFloatFlags.ieee
-                ^ ((bits == 80) ? CustomFloatFlags.storeNormalized : CustomFloatFlags.none);
+                ^ ((bits == 80 && !isIEEEQuadruple) ? CustomFloatFlags.storeNormalized : CustomFloatFlags.none);
```

Comment:
I'm pretty sure this is not quite right when going from 64bit -> 128bit CustomFloats, because the biases differ, you'll get a different exponent out than what you got in without any proper fix-up.

Reply:
Oh wait, the fault is mine, ignore. :-)

Reply:
Sorry, I don't fully understand, could you please explain it further? Is there anything to do with 64bit floats? I thought this line was only a hack of 80bit real type (which is actually defined quite differently from others). By the way, I can verify that without this judgment it still won't work; and I think the quadruple precision real type itself should be storeNormalized.

Reply:
The problem I was running into was bitfields fault.  Originally my fields on ppc64be were declared in the order I'd expect them to be in, but due to std.bitfields not being endian-aware, it instead generated the code as-if the order were reversed.